readme/Installation.scalatex

```diff
@@ -90,8 +90,13 @@
     There are several ways to install the CLI.
 
     @sect{Linux/OSX}
+    
+      @sect{Arch Linux}
```

Comment:
It appears a scalatex section can't immediately start with another section. Could you move this subsection after the `curl` instructions?